sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/FileFormatWriter.scala

```diff
@@ -307,6 +307,7 @@ object FileFormatWriter extends Logging {
           dataWriter.writeWithIterator(iterator)
           dataWriter.commit()
         }
+        description.statsTrackers.foreach(_.updateTaskWriteAndCommitDuration(timeCost))
```

Comment:
I should have pointed out this earlier: I don't think it makes sense to report the write time, as it can be misleading. Consuming the iterator also includes the time of executing the previous SQL operators.

Can we only track the task commit time?

Reply:
> I should have pointed out this earlier: I don't think it makes sense to report the write time, as it can be misleading. Consuming the iterator also includes the time of executing the previous SQL operators.
> 
> Can we only track the task commit time?

How about current?